<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Cadastro</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="nome_label">
              <property name="text">
               <string>Nome</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="inputNome"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="idade">
              <property name="text">
               <string>Idade</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="inputIdade"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnCadastrar">
            <property name="text">
             <string>Cadastrar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tabela">
            <column>
             <property name="text">
              <string>Nome</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Idade</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Categoria</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnIdade">
            <property name="text">
             <string>Ordenar por idade</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnNome">
            <property name="text">
             <string>Ordenar por nome</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Sortear">
            <property name="text">
             <string>Sortear</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Estatisticas</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>10</y>
            <width>511</width>
            <height>281</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="Ordena_Maioridade">
             <property name="title">
              <string>Maior Idade</string>
             </property>
             <widget class="QLabel" name="outMaiorIdade">
              <property name="geometry">
               <rect>
                <x>190</x>
                <y>30</y>
                <width>121</width>
                <height>41</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="ordenar_Menoridade">
             <property name="title">
              <string>Menor Idade</string>
             </property>
             <widget class="QLabel" name="outMenorIdade">
              <property name="geometry">
               <rect>
                <x>210</x>
                <y>40</y>
                <width>101</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="media">
             <property name="title">
              <string>Media das Idades</string>
             </property>
             <widget class="QLabel" name="outMediaIdade">
              <property name="geometry">
               <rect>
                <x>200</x>
                <y>30</y>
                <width>111</width>
                <height>31</height>
               </rect>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>359</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionCarregar"/>
    <addaction name="actionSalvar"/>
    <addaction name="separator"/>
    <addaction name="actionLimpar_Tabela"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLimpar_Tabela">
   <property name="text">
    <string>Limpar Tabela</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
